command-expansion-server/src/app.ts

```diff
@@ -14,7 +14,8 @@ import { commandDictionaryTypescriptBatchLoader } from './lib/batchLoaders/comma
 import { activitySchemaBatchLoader } from './lib/batchLoaders/activitySchemaBatchLoader.js';
```

Comment:
:large_blue_circle: Seems like this commit has the message that should go with some of the code that enters the code base in [ff6fdf](https://github.com/NASA-AMMOS/aerie/pull/166/commits/ff6fdf9141556131dcc0b6081fb93b360dc171c5)